### ISSUE TITLE
508-defect-2 [COGNITION]: Links MUST NOT rely on color

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18705]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Details

Links must not rely on color. Underlines are provided only on hover, which is not available to mobile users.

## Acceptance Criteria

- [ ] Links do not rely on color to convey that they can be interacted with 

## Proposed Solution (

[As discussed in ticket #3556, these heading links should be underlined](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556).

## WCAG or Vendor Guidance

* [WCAG 1.4.1: Use of Color](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)

### With color
<img width="833" alt="Screen Shot 2021-01-18 at 5 49 57 PM" src="https://user-images.githubusercontent.com/14154792/104969199-cee50400-59b5-11eb-95b2-f527d6499a27.png">

### Without color
<img width="975" alt="Screen Shot 2021-01-18 at 5 50 41 PM" src="https://user-images.githubusercontent.com/14154792/104969201-d0aec780-59b5-11eb-93cb-edb70e3031ee.png">

**Comment by DanielleThierryUSDSVA** [comment]
@SKasimow @youngfreezyVA This is another sitewide Drupal component issue.

**Comment by SKasimow** [comment]
@youngfreezyVA - to check if any thing is being developed to address site-wide

**Comment by joshkimux** [comment]
@SKasimow @youngfreezyVA https://github.com/department-of-veterans-affairs/vets-website/pull/15793
This might be a ticket worth checking out!

**Comment by youngfreezyVA** [comment]
ah yeah thanks @joshkimux looks like it's being addressed with that PR
